packages/web-react/src/hooks/useClickOutside.ts

```diff
@@ -0,0 +1,21 @@
+import { useEffect, MutableRefObject } from 'react';
+
+export interface UseClickOutsideProps {
+  ref: MutableRefObject<HTMLElement | undefined>;
+  callback?: (event: Event) => void;
+}
+
+export const useClickOutside = ({ ref, callback }: UseClickOutsideProps): void => {
+  const clickHandler = (event: Event) => {
+    if (!ref?.current?.contains(event?.target as Node)) {
+      if (callback) callback(event);
```

Comment:
I think this should be one condition `!ref?.current?.contains(event?.target as Node) && callback`, shouldn't it?

Reply:
Ok, I agree, fixed